### PR TITLE
Footer/ TypeTag/ NavBanner Component

### LINKED ISSUE #15: component/TypeTag
Image

### LINKED ISSUE #19: component/NavBanner
Image

### LINKED ISSUE #21: component/Footer
Image

### PR DESCRIPTION
**Pull request by JohnnyHsu041**
Related issues:
resolved #15 
resolved #19 
resolved #21 

1. 建立Footer, NavBanner, TypeTag component
2. 修改Button component在Card.tsx的file path
3. 修改logo png大小(1.5x)及檔名；更新有引用圖檔的路徑

### CODE REVIEW

**Review by muchuanhung — APPROVED** [review]
ok 已新增footer 待併入整合

**Comment by wen1011** [comment]
截圖 2024-05-23 上午10 12 39
可以跑一下 npm run build ~u報錯

**Review by viss5vuus98 — APPROVED** [review]

**Review by wen1011 — APPROVED** [review]
Has conflicts that must be resolved～ 解完就可以Merge了

File: src/components/layout/Footer/index.tsx
[the remark this answers is not in the document]
Comment (JohnnyHsu041) [reply src/components/layout/Footer/index.tsx:1]: 只是標註名稱而已

**Pull request merged**